tests/platform/broadcom/files/ser_injector.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
+
+import subprocess
+import re
+import time
+
+# Global parameter for memory scanners
+MEMORY_SCAN_INTERVAL_USEC = int(3e5)
+MEMORY_SCAN_ENTRIES = 16384
+SRAM_SCAN_INTERVAL_USEC = int(3e5)
+SRAM_SCAN_ENTRIES = 16384
+
```

Comment:
number of entries are the same for all BRCM chips we support?

Reply:
Those are SER thread specific and defines the number of entries to be scanned during one run of the thread. So, I assume they are chip type agnostic